velox/tpch/gen/dbgen/build.cpp

```diff
@@ -109,18 +113,36 @@ void init_build_buffers() {
   char** mk_ascdate PROTO((void));
   asc_date = mk_ascdate();
 
-  sprintf(orderSzFormat, O_CLRK_FMT, 9, &HUGE_FORMAT[1]);
-  sprintf(custSzFormat, C_NAME_FMT, 9, &HUGE_FORMAT[1]);
-  sprintf(partSzFormat, P_MFG_FMT, 1, &HUGE_FORMAT[1]);
-  sprintf(partSzBrandFormat, P_BRND_FMT, 2, &HUGE_FORMAT[1]);
-  sprintf(suppSzFormat, S_NAME_FMT, 9, &HUGE_FORMAT[1]);
+  auto res = sprintf(orderSzFormat, O_CLRK_FMT, 9, &HUGE_FORMAT[1]);
```

Comment:
It should be okay to initialize each of the char buffers above with 0 and remove the checks here.
That should avoid folly dependency.

Reply:
Done.